action.yml

```diff
@@ -114,7 +114,7 @@ runs:
 
         if [ -n "${{ inputs.extra-conf }}" ]; then
           if [ -n "${{ inputs.github-token }}" ]; then
-            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}\naccess-tokens = github.com=${{ inputs.github-token }}"
+            export NIX_INSTALLER_EXTRA_CONF="${{ inputs.extra-conf }}access-tokens = github.com=${{ inputs.github-token }}"
```

Comment:
Hmm I am wondering if there are cases where we need to include a newline, eg if the user enters `extra-conf: "foobar"` with no newline?

Reply:
We do not, it seems fine!